static/third/spectrum/spectrum.js

```diff
@@ -132,7 +132,7 @@
         return opts;
     }
 
-    function spectrum(element, o) {
+    function spectrum(element, o, optionalSelector) {
```

Comment:
Could we move `optionalSelector` to be a proper option to `spectrum` rather than an additional argument?